components/Result.tsx

```diff
@@ -22,10 +23,11 @@ export default function Result({ album }: { album: Album }) {
         )}
       </div>
 
-      <div className="text-sm">
+      <div className="text-sm max-w-[250px]">
```

Comment:
this does feel a bit cluttered:

![CleanShot 2023-10-28 at 20 19 30](https://github.com/Rmaki27/collections/assets/37197876/1b449f71-ed59-482a-ba5a-4bb188126e4b)

up to you if you want to change it, but my suggestions would be:
- add margin-top
- remove this max w or maybe make it bigger maybe

Reply:
Moved max width to title and artist div, removed max width from description, added mt to description